python/ray/experimental/dask/common.py

```diff
@@ -0,0 +1,77 @@
+from collections import OrderedDict
```

Comment:
Lmk if y'all would rather this unpacking/repacking utility not be broken out into its own module. This being in a separate module is primarily an artifact of it being reused across a few scheduler plugins in our monorepo; if those plugins are deemed useful enough for upstreaming, then it may make sense to keep this in a separate module.

Reply:
Having it in dask/common.py as is right now seems fine to me.